rebuild.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
 export CFLAGS="-Wall -Wextra -Wdeclaration-after-statement -Wmissing-field-initializers -Wshadow -Wno-unused-parameter -ggdb3"
-phpize && ./configure --with-mongo-sasl && make clean && make -j 5 all && make install
+phpize > /dev/null && \
+./configure --with-mongo-sasl > /dev/null && \
+make clean && make -j 9 all > /dev/null && make install
```

Comment:
I think the "-j 9" part should go, as you can set the ```MAKEFLAGS='-j 20'``` in ``.bashrc`` too. I don't want to be restricted to only 9 cores ;-þ